### PR TITLE
specifying the path for global install, follow links, actually global…

### PR DESCRIPTION
**Pull request by bruce-au**
…ly installing

### CODE REVIEW

File: git-hooks [hunk 1]
```diff
@@ -182,31 +182,34 @@ git-hooks run "$0" "$@"';
     fi
 }
 
-function install_global 
+function install_global
 {
     TEMPLATE="$HOME/.git-template-with-git-hooks"
+    if [[ -n "$1" ]]; then
```
Comment (misatos) [inline git-hooks]: so this allows us to use our own template?

File: git-hooks [hunk 2]
```diff
@@ -226,7 +229,7 @@ case $1 in
         install_hooks "$1"
         ;;
     --install-global|--installglobal )
-        install_global
+        install_global "$2"
```
Comment (misatos) [inline git-hooks]: what's in $2?

File: git-hooks [hunk 3]
```diff
@@ -61,7 +61,7 @@ function list_hooks_in_dir
 	 -mindepth ${level} \
 	 -maxdepth ${level} \
 	 \( -perm -100 -o -perm -010 -o -perm -001 \) \
-	 -type f 2>/dev/null \
+	 -type fl 2>/dev/null \
```
Comment (misatos) [inline git-hooks]: what does this change do?

File: git-hooks (hunk 1 above)
Comment (bruce-au) [reply git-hooks]: This allows us to specify where the template directory is being created

File: git-hooks (hunk 2 above)
Comment (bruce-au) [reply git-hooks]: We're expecting the path to the template repository to be created

File: git-hooks (hunk 3 above)
Comment (bruce-au) [reply git-hooks]: this allows find to return both symlinks and files

File: git-hooks (hunk 2 above)
Comment (misatos) [reply git-hooks]: Is it safer to check that there's a value in $2?

Comment (bruce-au) [reply git-hooks]: https://github.com/autonomic-ai/git-hooks/pull/1/files#diff-b4570c984375f2d3c836fdf385384c4eR188 We're checking for empty string in here

**Pull request merged**